Assets/Plugins/Source/Editor/EOSPluginEditorConfig.cs

```diff
@@ -167,6 +167,61 @@ public void SaveToJSONConfig(bool prettyPrint)
         }
     }
 
+
+    public class EOSPluginEditorVersionConfigSection : IEOSPluginEditorConfigurationSection
+    {
+        private static string ConfigName = "eos_plugin_version_config.json";
+        private EOSConfigFile<EOSPluginEditorVersionConfig> configFile;
+
+        [InitializeOnLoadMethod]
+        static void Register()
+        {
+            EOSPluginEditorConfigEditor.AddConfigurationSectionEditor(new EOSPluginEditorVersionConfigSection());
+        }
+
+        //-------------------------------------------------------------------------
+        public string GetNameForMenu()
+        {
+            return "Prebuild Settings";
+        }
+
+        //-------------------------------------------------------------------------
+        public void Awake()
+        {
+            var configFilenamePath = EOSPluginEditorConfigEditor.GetConfigPath(ConfigName);
+            configFile = new EOSConfigFile<EOSPluginEditorVersionConfig>(configFilenamePath);
+        }
+
+        //-------------------------------------------------------------------------
+        public bool DoesHaveUnsavedChanges()
+        {
+            return false;
+        }
+
+        //-------------------------------------------------------------------------
+        public void LoadConfigFromDisk()
+        {
+            configFile.LoadConfigFromDisk();
+        }
+
+        public EOSPluginEditorVersionConfig GetCurrentConfig()
+        {
+            return configFile.currentEOSConfig;
+        }
+
+        //-------------------------------------------------------------------------
+        void IEOSPluginEditorConfigurationSection.OnGUI()
+        {
+            EpicOnlineServicesConfigEditor.AssigningBoolField("Use app version as product version", ref configFile.currentEOSConfig.useAppVersionAsProductVersion);
```

Comment:
I would suggest a more specific wording, such as : "Use Unity App Version for the EOS product version"